examples/apps/weather/src/Display.flix

```diff
@@ -0,0 +1,20 @@
+mod Display {
+    use Location.Loc
+    use Weather.WeatherInfo
+
+    pub eff Display {
+        def display(loc: Loc, weather: WeatherInfo): Unit
+    }
+
+    ///
+    /// Handles display effect by printing to stdout
+    ///
+    pub def displayWithIO(f: Unit -> a \ ef): a \ ef - Display + IO =
+        run {
+            f()
+        } with handler Display {
+            def display(loc, weather, resume) = 
+                println("Weather for ${loc}:\n\t ${weather}") |> resume
```

Comment:
This line seems a little strange. You're piping the result of `println`, which is `Unit`.

I think it's a little clearer if you just do
```
println(...);
resume()
```